backend/cdApp.py

```
@@ -161,15 +170,17 @@ def get(self, course_id):
 class Root(Resource):
     def post(self, course_id, operation, root_id):
         if operation == 'set':
-            g.db.execute('''UPDATE nodes 
-                            SET isroot=1 
-                            WHERE id=%s AND course_id=%s AND isalive=1''' % (root_id, course_id))
+            g.db.execute('''UPDATE nodes
+                            SET isroot=1
+                            WHERE id=(?) AND course_id=(?) AND isalive=1''',
+                         [int(root_id), int(course_id)])
```

Good call. Right now, it silently fails. I can fix this. Will push in a second.